lib/vindata/services/nada.rb

```diff
@@ -33,13 +33,20 @@ def get_token
 
       wsdl_path = File.expand_path(File.join(File.dirname(__FILE__), '../wsdls/ProdSecureLogin.wsdl'))
       # do auth login to get token
+      ssl_cert = File.join(__dir__, 'nada_certs/TrustedRoot.pem')
```

Comment:
I'm not sure this is going to work, since you are inside `lib/vidata/services/`, instead you should do: 
File.expand_path(File.join(__dir__, '../../nada_certs/TrustedRoot.pem'))